spectrum.css

```diff
@@ -98,6 +110,21 @@ License: MIT
     left: 0;
     height: 8px;
 }
+
```

Comment:
There's a rule in this file for `.sp-alpha-enabled .sp-top` that sets a margin bottom.  We'd need to override that and force the same rule that applies margin-bottom to 3px for .sp-top in this case